docs/transparent-validation/muxpi/theory-of-operations.md

```diff
@@ -91,7 +93,7 @@ managed=false
 
 Add network setting to `/etc/network/interfaces`:
 
-`sudo vi /etc/network/interfaces`
+    sudo vi /etc/network/interfaces
```

Comment:
Code blocks should be added as follows:

```bash
<insert Your code here>
```
Please refer to another files in documentation.